pkgs/applications/editors/vscode/extensions/python/default.nix

```
@@ -57,14 +57,16 @@ vscode-utils.buildVscodeMarketplaceExtension rec {
       curl
       coreutils
       gnused
+      jq
       nix
     ]}
 
     api=$(curl -s 'https://marketplace.visualstudio.com/_apis/public/gallery/extensionquery' \
       -H 'accept: application/json;api-version=3.0-preview.1' \
       -H 'content-type: application/json' \
-      --data-raw '{"filters":[{"criteria":[{"filterType":7,"value":"${mktplcRef.publisher}.${mktplcRef.name}"}]}],"flags":512}')
-    version=$(echo $api | sed -n -E 's|^.*"version":"([0-9.]+)".*$|\1|p')
+      --data-raw '{"filters":[{"criteria":[{"filterType":7,"value":"${mktplcRef.publisher}.${mktplcRef.name}"}]}],"flags":16}')
+    # Find the latest version compatial with stable vscode version
```

```suggestion
    # Find the latest version compatible with stable vscode version
```
compatial => compatible